Dockerfile.api

```diff
@@ -1,4 +1,6 @@
 FROM golang:1.8
+RUN apt update; \
+    apt-get -y install xmlsec1;
```

Comment:
External program needed for SAML implementation. Need to verify this is present in deployed environments.